charts/tesk/templates/ingress/default-server-secret.yaml

```diff
@@ -0,0 +1,10 @@
+{{ if and (eq .Values.clusterType "kubernetes") .Values.kubernetes.deploy_ingress }}
```

Comment:
Bit confused, what this if loop does

Reply:
Pls add a comment in yaml

Reply:
Not a loop, but a double condition. It is confusing because it is prefix, but in infix is:

if ((.Values.clusterType == 'Kubernetes') and .Values.kubernetes.deploy_ingress == True )

Reply:
Added

Reply:
thnx!